src/uct/ib/rc/verbs/rc_verbs_ep.c

```diff
@@ -269,13 +271,15 @@ ucs_status_t uct_rc_verbs_ep_get_bcopy(uct_ep_h tl_ep,
     return UCS_INPROGRESS;
 }
 
-ucs_status_t uct_rc_verbs_ep_get_zcopy(uct_ep_h tl_ep, void *buffer, size_t length,
-                                       uct_mem_h memh, uint64_t remote_addr,
-                                       uct_rkey_t rkey, uct_completion_t *comp)
+ucs_status_t uct_rc_verbs_ep_get_zcopy(uct_ep_h tl_ep, const uct_iov_t *iov, size_t size,
+                                       uint64_t remote_addr, uct_rkey_t rkey,
+                                       uct_completion_t *comp)
 {
     uct_rc_verbs_ep_t *ep = ucs_derived_of(tl_ep, uct_rc_verbs_ep_t);
     ucs_status_t status;
 
+    UCT_CHECK_PARAM_IOV;
```

Comment:
@MattBBaker Could you please look into this (for example) function change. I need to do this with ugni transport but has no machine to compile it. Could you please help?